src/components/WalletNameStep/WalletNameStep.js

```diff
@@ -20,26 +20,28 @@ const WalletNameStep = ({
   valueName,
   isLoading,
 }: Props) => (
-  <form className='wallet-name-step' onSubmit={ignoreEvent(onSubmit)()}>
-    <JInput
-      onChange={onChangeName}
-      value={valueName}
-      errorMessage={invalidFields.name}
-      color='white'
-      placeholder='Wallet name'
-      name='wallets-create-name'
-    />
-    <div className='actions'>
-      <JRaisedButton
-        onClick={onSubmit}
-        color='blue'
-        label='Next step'
-        loaderColor='white'
-        isLoading={isLoading}
-        isWide
+  <div className='wallet-name-step'>
+    <form className='form' onSubmit={ignoreEvent(onSubmit)()}>
+      <JInput
+        onChange={onChangeName}
+        value={valueName}
+        errorMessage={invalidFields.name}
+        color='white'
+        placeholder='Wallet name'
+        name='wallets-create-name'
```

Comment:
Please check this file [on feature/update-import-wallet branch](https://github.com/jibrelnetwork/jwallet-web/blob/feature/update-import-wallet/src/components/WalletNameStep/WalletNameStep.js)
as well as WalletPasswordStep
there are some new changes, like placeholder values. Would be better to apply them here.